migration/migration-technical-details.md

```diff
@@ -51,12 +55,25 @@ After the pre-staking phase, a clear overview of the registered validators and t
 
 ## Activation Phase
 
-Only pre-registered validators can participate in this phase. The activation phase serves two primary purposes:
+Only pre-registered validators can participate in this phase. The activation phase serves the purpose of executing the transition. At the first candidate block's height (block 3 456 000), if at least 80% of the total stake signals readiness for the transition, the candidate block becomes the transition block. A special activation tool captures and migrates the state of the PoW chain at the transition block, including the transaction history, to generate the "genesis block" for the new PoS chain.
+
+The Activation Phase follows a specific sequence of events:
+1. The candidate block is mined at block height 3 456 000, which opens the first activation window.
+2. The network waits for 10 confirmations to prevent block reversion.
+3. Once 10 confirmation blocks are mined, registered validators capture the state of block 3 456 000 and start generating the "genesis" block.
+4. Validators generate the "genesis" block deterministically. The block hash should be the same across all validators.
+5. After generating the "genesis" block, validators send their readiness transactions, which include the block hash in the data field.
+6. The network monitors readiness transactions within a 24 hour window.
+7. If 80% of the total stake (represented by registered validators) sends readiness transactions within 24 hours, the PoS chain starts at block height 3 456 000. If the threshold is not reached, the process automatically moves to the next activation window.
+8. If the 80% readiness threshold is not met within the first 24 hours, a new activation window begins, starting with a new candidate block at the block immediately following 1440 blocks or 24 hours from the first candidate block.
+9. In subsequent activation windows, validators regenerate the "genesis" block (3 457 440), send readiness transactions again, and the process repeats until 80% readiness is achieved.
```

Comment:
In subsequent activation windows, the same exact process is repeated. Selecting a new candidate block that is 24 hours apart from the previous one. Until the PoS chain is started